### PR TITLE
Implementation of movie information

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
Changes of last PR:
* Implementation of generic bloc interface.

I attach an image of movie card information and a gif that show how to go to that.
image

movieInfoGif

### CODE REVIEW

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by MarcosCGlobant — APPROVED** [review]

File: lib/src/styles/dimensions.dart [hunk 1]
```diff
@@ -8,4 +8,29 @@ abstract class Dimension {
   static const double cardBorderRadius = 8.0;
   static const double clipBorderRadius = 8.0;
   static const double cardContainerEdgeInsets = 5.0;
+  static const double RatingContainerHeight = 60;
```
Comment (luisjavierglb) [inline lib/src/styles/dimensions.dart:11]: These variables shoud be in camel case as well

**Review by luisjavierglb — APPROVED** [review]
LGTM

File: lib/src/styles/dimensions.dart (hunk 1 above)
Comment (FranciscoGaraventa) [reply lib/src/styles/dimensions.dart:11]: Solved.

**Pull request merged**